packages/babel-preset-udemy-website/index.js

```diff
@@ -30,6 +29,7 @@ module.exports = declare((api, options) => {
                     },
                     modules: false,
                     useBuiltIns: 'entry',
+                    corejs: 2,
```

Comment:
babel complains if I don't set this. babel now supports core-js v3, so they make you set this option now to say which version of core-js you're going to use.

Reply:
what's the difference between 2 and 3? i.e. is upgrading to 3 worth it?

Reply:
I made a card for us to look into it: https://udemyjira.atlassian.net/browse/TF-3780
Basically it looks like we don't have to upgrade until we want to start using a JS feature that core-js 2 doesn't support. All new features will be implemented in core-js 3 only. The feature set for core-js 2 has been frozen.